apis/r/tools/build_libtiledbsoma.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+## This script is invoked from the tarball-getting script `get_tarballs.R` which
+## itself is called from `configure`.
+
+cwd=`pwd`
+
+## This helps a little
+mv inst/tiledbsoma/TileDB-SOMA-* inst/tiledbsoma/TileDB-SOMA
+
+## Now make build/ and call cmake; make; make install
+mkdir inst/tiledbsoma/TileDB-SOMA/libtiledbsoma/build && \
+    cd inst/tiledbsoma/TileDB-SOMA/libtiledbsoma/build && \
+    cmake -DDOWNLOAD_TILEDB_PREBUILT=ON \
```

Comment:
Does `-DDOWNLOAD_TILEDB_PREBUILT=ON` break the developer experience for anyone (many of us within TileDB for example) who do have core checked out?